utils/generate_rules_import.py

```diff
@@ -7,7 +7,11 @@
 
 # import parrent rule class
 rules_pkg_path = "validator.rules_src"
-line = f"from {rules_pkg_path} import Rule\n\n"
+line = f"from {rules_pkg_path} import Rule\n"
+rules_file.writelines(line)
+
+# import __all__ object
+line = f"from {rules_pkg_path} import __all__ \n"
```

Comment:
There is trailing whitespace between __all__ and \n. That's why error occurred. Please fix it